octobot_trading/exchanges/exchange_manager.py

```diff
@@ -218,6 +218,12 @@ async def _init_simulated_exchange(self):
     Rest exchange
     """
 
+    async def _search_and_create_spot_or_rest_exchange(self):
+        self.exchange = None
+        await self._search_and_create_spot_exchange()
```

Comment:
It's duplicating code. It should be done in `_create_real_exchange` with : 
```
# create REST based on ccxt exchange
        if self.is_future and not self.is_spot_only:
            await self._search_and_create_future_exchange()
        elif self.is_margin and not self.is_spot_only:
            await self._search_and_create_margin_exchange()
         if self.is_spot_only or self.exchange is None:
            await self._search_and_create_spot_exchange()

        if not self.exchange:
            await self._search_and_create_rest_exchange()
```
If we really want this behaviour